packages/components/pie-checkbox-group/test/component/pie-checkbox-group.spec.ts

```diff
@@ -17,4 +33,101 @@ test.describe('PieCheckboxGroup - Component tests', () => {
+            // Arrange
+            await mount(PieCheckboxGroup, {
+                props: {
+                    assistiveText: 'Assistive text',
+                } as PieCheckboxGroup,
+            });
+
+            // Act
+            const assistiveText = page.locator(assistiveTextSelector);
+
+            // Assert
+            expect(assistiveText).toBeVisible();
+            expect(await assistiveText.getAttribute('variant')).toBe('default');
+            expect(assistiveText).toHaveText('Assistive text');
+        });
+
+        test.describe('Assistive text: Status', () => {
+            statusTypes.forEach((status) => {
+                test(`should render the assistive text component with the ${status} variant`, async ({ mount, page }) => {
+                    // Arrange
+                    await mount(PieCheckboxGroup, {
+                        props: {
+                            assistiveText: 'Assistive text',
+                            status,
+                        } as PieCheckboxGroup,
+                    });
+
+                    // Act
+                    const assistiveText = page.locator(assistiveTextSelector);
+
+                    // Assert
+                    expect(assistiveText).toBeVisible();
+                    expect(assistiveText).toHaveAttribute('variant', status);
+                    expect(assistiveText).toHaveText('Assistive text');
+                });
+            });
+        });
+
+        test.describe('Assistive test ID attribute', () => {
+            test('should contain an ID associated with the checkbox group element for a11y', async ({ mount, page }) => {
+                // Arrange
+                const component = await mount(PieCheckboxGroup, {
+                    props: {
+                        assistiveText: 'Assistive text',
+                    } as PieCheckboxGroup,
+                });
+
+                // Act
+                const checkboxGroup = component.locator(componentSelector);
+                const assistiveText = page.locator(assistiveTextSelector);
+
+                const componentAttribute = await checkboxGroup.getAttribute('aria-describedby');
+
+                // Assert
+                await expect(assistiveText).toHaveAttribute('id', 'assistive-text');
+                expect(componentAttribute).toBe('assistive-text');
+            });
+        });
+    });
+
+    test.describe('Props', () => {
+        test.describe('disabled', () => {
+            test.describe('when true', () => {
+                test('should disable the slotted input component', async ({ mount }) => {
+                    // Arrange
+                    const component = await mount(PieCheckboxGroup, {
+                        props: {
+                            disabled: true,
+                        } as CheckboxGroupProps,
+                        slots: {
+                            default: '<input type="checkbox" data-test-id="test-checkbox" />',
```

Comment:
I think testing with `pie-checkbox` would be good here too. And perhaps checking that individual checkboxes can still be disabled even when `pie-checkbox-group`'s `disabled` prop is false?